target/linux/ramips/image/mt7621.mk

```diff
@@ -708,8 +708,9 @@ define Device/MikroTik
   BLOCKSIZE := 64k
   IMAGE_SIZE := 16128k
   DEVICE_PACKAGES := kmod-usb3
-  LOADER_TYPE := elf
-  KERNEL := $(KERNEL_DTB) | loader-kernel
+  KERNEL_NAME := vmlinux.elf
+  KERNEL_NAME := vmlinux-initramfs.elf
```

Comment:
This could be skipped. Just use vmlinuz directly.
As you've experienced before, using vmlinux will get current kernel memory detection code confused.